test/groovy/DockerExecuteTest.groovy

```diff
@@ -1,21 +1,14 @@
-
+import com.sap.piper.JenkinsUtils
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
+import util.*
 
-import util.BasePiperTest
-import util.JenkinsLoggingRule
-import util.Rules
-import util.JenkinsStepRule
-
-import static org.junit.Assert.assertEquals
-import static org.junit.Assert.assertTrue
-import static org.junit.Assert.assertFalse
+import static org.junit.Assert.*
```

Comment:
I vote for keeping all methods enumerted instead of having the * here.

Reply:
Thanks to IntelliJ formatting. I will revert it